tests/tests.js

```diff
@@ -1,45 +1,141 @@
     }
-    const price = pricing.calculateProductPrice(products.longTermDisability, employee, selectedOptions)
-
-    expect(price).to.equal(22.04)
+    expect(commuterParking).to.equal(175)
   })
+})
+describe('getEmployerContribution', () => {})
+it('returns a truncated price with two decimal places not rounded when given a number with multiple places', () => {})
 
-  it('throws an error on unknown product type', () => {
-    const unknownProduct = { type: 'vision' }
+describe('calculatesLTDPrice', () => {
+      it('returns the LTD price minus the employer contribution', () => {
+        const selectedOptions = {
+          familyMembersToCover: [ee]
+        }
+        const price = pricing.calculateLTDPrice(products.voluntaryLife, employee, selectedOptions)
+      })
 
-    expect(() => pricing.calculateProductPrice(unknownProduct, {}, {})).to.throw('Unknown product type: vision')
-  })
-})
+      describe('calculateProductPrice', () => {
+            let calculateVolLifePriceSpy
+            let getEmployerContributionSpy
+            let formatPriceSpy
+            let calculateLTDPriceSpy
+            let sandbox
+
+            beforeEach(() => {
+              sandbox = sinon.createSandbox()
+
+              calculateVolLifePriceSpy = sandbox.spy(pricing, 'calculateVolLifePrice')
+              getEmployerContributionSpy = sandbox.spy(pricing, 'getEmployerContribution')
+              formatPriceSpy = sandbox.spy(pricing, 'formatPrice')
+              calculateLTDPriceSpy = sandbox.spy(pricing, 'calculateLTDPrice')
+            })
+            afterEach(() => {
+              sandbox.restore()
+            })
+
+            it('returns the price for a voluntary life product for a single employee', () => {
+              const selectedOptions = {
+                familyMembersToCover: ['ee'],
+                coverageLevel: [{
+                  role: 'ee',
+                  coverage: 125000
+                }],
+              }
+              const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+              expect(price).to.equal(39.37)
+              expect(calculateVolLifePriceSpy).to.have.callCounts(1)
+              expect(getEmployerContributionSpy).to.have.callCounts(1)
+              expect(formatPriceSpy).to.have.callCounts(1)
+              expect(calculateLTDPriceSpy).to.have.callCount(0)
+            })
+
+            describe('calculateProductPrice', () => {
+              it('returns the price for a voluntary life product for a single employee', () => {
+                const selectedOptions = {
+                  familyMembersToCover: ['ee'],
+                  coverageLevel: [{
+                    role: 'ee',
+                    coverage: 125000
+                  }],
+                }
+                const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+
+                expect(price).to.equal(39.37)
+              })
```

Comment:
This is a duplicate of the test right above it but this one doesn't have the spies which makes it a bad unit test instead of a good integration test like that one.